recipe/meta.yaml

```diff
@@ -1,37 +1,37 @@
-{% set version = "3.1.2" %}
+{% set version = "3.1.3" %}
 
 package:
   name: cdtime
   version: {{ version }}
 
 source:
   url: https://github.com/cdat/cdtime/archive/v{{ version }}.tar.gz
-  sha256: e6aa613bb509a67f8dc0050d68ffc86c33099e19cfb2b64374b53131b9646e52
+  sha256: 121b9e855c3889d145bc9730d36403df6fd1b7677e99076556e763c3e53a59d5
 
 build:
-  number: 8
+  number: 0
   skip: True  # [win]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('fortran') }}
-
   host:
     - python
     - setuptools
+    - cython
     - libcdms
     - numpy
     - libnetcdf
-    - cdat_info >=8.1.1
+    - cdat_info
```

Comment:
@muryanto1, does this mean that this latest release works with any version of cdat_info?  If not, it's still helpful to have a restriction here.  Most often, you'll pick up the latest build of the latest release anyway but the constraints can help protect you from known incompatibilities where they exist.